jvm/src/main/kotlin/org/bibletranslationtools/maui/jvm/ui/main/MainViewModel.kt

```diff
@@ -58,6 +71,31 @@ class MainViewModel : ViewModel() {
         importFiles(filesToImport)
     }
 
+    fun verify() {
+        val directoryChooser = DirectoryChooser()
+        val file = directoryChooser.showDialog(primaryStage)
+        val filename = "${file.absolutePath}/report.html"
+
+        isProcessing.set(true)
+        fileDataList.toRxObservable()
+            .map { fileDataItem ->
+                fileVerifier.handleItem(fileDataItem)
+            }
+            .toList()
+            .map { results ->
+                verifiedResultMapper.fromEntity(results)
+            }
+            .map { context ->
+                thymeleafEngine.process("VerifiedResultsTemplate", context)
+            }
+            .observeOnFx()
+            .doFinally { isProcessing.set(false) }
+            .subscribe { html ->
+                docWriter.write(filename, html)
```

Comment:
when I see `docWriter`, I think about _.doc_ or _.docx_ instead of html. Could we rename the class and variable to HtmlWriter?